app/javascript/components/supplemental_application/actions.js

```diff
@@ -1,18 +1,35 @@
 import apiService from '~/apiService'
 import domainToApi from '~/components/mappers/domainToApi'
 import Alerts from '~/components/Alerts'
-import { isEmpty } from 'lodash'
+import { isEmpty, find, isEqual, every } from 'lodash'
 
-export const updateApplication = async (application) => {
-  const leasePromise = updateLease(
-    application['lease'], application['primaryApplicantContact'], application['id']
-  )
+export const updateApplication = async (application, prevApplication) => {
```

Comment:
@software-project This looks like a different approach from saving unsaved preferences when saving the whole application, do you see the way it was currently implemented, it looks like the save preference feature got lost.

Reply:
Yes, it's different as preferences are saved with suppl application via salesforce api and @akegan didn't want to modify salesforce for this one. Preferences are saved as usual.